src/routes/tools/slr-coastal-inundation/_Map/Rasters.svelte

```diff
@@ -104,10 +108,10 @@
   }
 
   onDestroy(() => {
-    map.off("styledata", handleStyleDataChange);
     removeRasterLayers();
     removePreviousRasterLayers();
     layerHandler.removeMapRef();
     layerHandler = null;
+    map.off("styledata", handleStyleDataChange);
```

Comment:
Do you want to move this map.off to the bottom of onDestroy in TileIndexes and TileCentroids as well?

Reply:
It probably doesn't matter either way but might be good to be consistent.

Reply:
Yeah agreed but get style points!